stock_prices/stock_prices.py

```diff
@@ -3,7 +3,16 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  profits = []
+  for i in enumerate(prices):
```

Comment:
As you are well aware of, enumerate returns a tuple of `(index, value)`. As an alternative to setting up a new variable to refer to the first element of the tuple, you can destructure right in the for loop:
```
  for index, value in enumerate(prices):
    # index = i[0] Not needed anymore, since index refers directly to the index integer
    if index < len(prices)-2:
       # so on and so forth
```